plugins/CoreHome/stylesheets/coreHome.less

```diff
@@ -9,7 +9,7 @@
 .card h2 {
     &.card-title,
     .card-title {
-        padding: 0 !important;
```

Comment:
Are you sure this won't have any side effects in core or for any plugin?

Reply:
I believe this is no effect from I can aware.

Reply:
revert that one